tests/suites/test_suite_psa_crypto_se_driver_hal.data

```diff
@@ -163,25 +163,25 @@ Key registration: key id max volatile
 register_key_smoke_test:TEST_SE_PERSISTENT_LIFETIME:7:PSA_KEY_ID_VOLATILE_MAX:1:PSA_ERROR_INVALID_HANDLE
 
 Import-sign-verify: sign in driver, ECDSA
-depends_on:MBEDTLS_ECDSA_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+depends_on:MBEDTLS_ECDSA_C:MBEDTLS_ECP_C:PSA_WANT_ECC_SECP_R1_256
```

Comment:
I am not sure why in fact but in the previous similar cases we didn't update the dependencies of test_suite_psa_crypto_se_driver_hal*.data files (see 8405b5f6). @gilles-peskine-arm why this exception ?

Reply:
I thought that the mock SE drivers in these tests might not work with a new-style accelerator driver. As long as the CI passes, and will keep passing when we improve coverage with accelerator drivers, it doesn't matter. Old-style SE support is currently on suspended animation: we don't try to improve it, but we do try not to break what's already there.

Reply:
ok thanks, Gilles. Thus I guess better to stay on the same path and do not update the dependencies in test_suite_psa_crypto_se_driver_hal.data with PSA_WANT* macros.